doc/introduction.rst

```diff
@@ -17,54 +17,114 @@
 .. You should have received a copy of the GNU General Public License
  along with OpenShot Library.  If not, see <http://www.gnu.org/licenses/>.
 
+.. Edited September 27, 2020 - [USA]TechDude
+
+User Guide
+==========
+
+This guide is a simple how-to to get the user started with enough information 
+to create a small, quick video by the end of the Quick Tutorial.  For more 
+in-depth information regarding |ops| OpenShot™, please refer to the 
+`User Manual <manual.html>`__.
+
+.. image:: images/title-ops.png
+
 Introduction
-============
+------------
 
-OpenShot Video Editor is an award-winning, open-source video editor, available on
-Linux, Mac, and Windows. OpenShot can create stunning videos, films, and animations with an
-easy-to-use interface and rich feature-set.
+Jonathan Thomas created |ops| OpenShot™ in 2008 to build a free, open-source 
+video editor for Linux.  We designed OpenShot™ to be easy to use, quick to 
+learn, and a powerful video editor.  OpenShot™ is available for the following 
+64-bit operating systems:
+
+ -  |win| `Microsoft® Windows® <install.html#win-windows>`_
+ -  |lin| `Linux® (most distributions) <install.html#lin-linux>`_
+ -  |mac| `Apple® macOS® <install.html#mac-macos>`_
+ 
+Users have downloaded OpenShot™ millions of times, and it continues to grow as 
+a project.  OpenShot™ Video Editor is an award-winning, open-source video 
+editor that can create stunning videos, films, and animations with an 
+easy-to-use interface and a rich feature-set.
 
 Features
 --------
-* Cross-platform (Supports Linux, OS X, and Windows)
-* Support for many video, audio, and image formats (based on FFmpeg)
-* Powerful curve-based Key frame animations
-* Desktop integration (drag and drop support)
-* Unlimited tracks / layers
-* Clip resizing, scaling, trimming, snapping, rotation, and cutting
-* Video transitions with real-time previews
-* Compositing, image overlays, watermarks
-* Title templates, title creation, sub-titles
-* 2D animation support (image sequences)
-* 3D animated titles (and effects)
-* SVG friendly, to create and include vector titles and credits
-* Scrolling motion picture credits
-* Advanced Timeline (including Drag & drop, scrolling, panning, zooming, and snapping)
-* Frame accuracy (step through each frame of video)
-* Time-mapping and speed changes on clips (slow/fast, forward/backward, etc...)
-* Audio mixing and editing
-* Digital video effects, including brightness, gamma, hue, greyscale, chroma key (bluescreen / greenscreen) , and many more!
+OpenShot™ Video Editor has many abilities and features, including:
```

Comment:
I was listing things from most important to least important...and not alphabetically. I still prefer we list the most useful / impressive / interesting bits first, and de-emphasize the less important bits.